Description: Use explicit version string.
 Instead of using package introspection to discover the version, just use
 an explicit version string.
 .
 moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium
 .
   * Initial release.
Author: Paul Boddie <paul@boddie.org.uk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-07-04

--- moin-2.0.orig/docs/conf.py
+++ moin-2.0/docs/conf.py
@@ -51,11 +51,10 @@ copyright = '2011-2022, The MoinMoin dev
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-from pkg_resources import get_distribution
 # The full version, including alpha/beta/rc tags.
-release = get_distribution('moin').version
+release = '2.0'
 # The short X.Y version.
-version = '.'.join(release.split('.')[:2])
+version = release
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
